internal/fakecgo/trampolines_stubs.s

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2022 The Ebitengine Authors
+
+//go:build darwin
+// +build darwin
+
+#include "textflag.h"
+
+// these stubs are here because it is not possible to go:linkname directly the C functions on darwin arm64
```

Comment:
Is this file needed only for darwin/arm64?

Reply:
No they are needed on both amd64 and arm64. The reason it has to be done this way is bc arm64 doesn't allow directly linking to dynamic C functions